### PR TITLE
Epoxy include in one header

### PR DESCRIPTION
**Pull request by acterhd**
Based on MSVC 2017 my commits. I preparing support with OpenGL glad or epoxy.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -42,7 +56,15 @@ if (CMAKE_VERSION VERSION_GREATER 2.8.5)
...
 endif ()
 
-if (APPLE)
+if (MSVC)
+  add_custom_target(versionInfo ALL
+    COMMAND del ${CMAKE_CURRENT_SOURCE_DIR}/source/Version.hpp
+    COMMAND echo -n \"\#define GAME_VERSION \\\"\">${CMAKE_CURRENT_SOURCE_DIR}/source/Version.hpp
+    COMMAND git describe | tr -d \"\\n\" >>${CMAKE_CURRENT_SOURCE_DIR}/source/Version.hpp
+    COMMAND echo -n \"\\\"\">>${CMAKE_CURRENT_SOURCE_DIR}/source/Version.hpp
+    COMMAND git update-index --assume-unchanged ${CMAKE_CURRENT_SOURCE_DIR}/source/Version.hpp
```
Comment (SGOrava) [inline CMakeLists.txt]: IIRC > Version.hpp should be in .gitigone

**Comment by acterhd** [comment]
Don't forget change submodules path, I changed for my workspace

File: .gitmodules [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/GlPortal/glportal-data.git
 [submodule "external/RadixEngine"]
 	path = external/RadixEngine
-	url = https://github.com/GlPortal/RadixEngine.git
+	url = https://github.com/acterhd/RadixEngine.git
```
Comment (ElementW) [inline .gitmodules]: Don't commit changes that point to an external repo.

**Comment by acterhd** [comment]
I reloaded fix for MSVC fully

**Pull request closed without merging**